src/elements/content-preview/preview-header/FileInfo.scss

```diff
@@ -0,0 +1,9 @@
+.bcpr-FileInfo {
+    align-items: center;
+    display: flex;
+}
+
+.bcpr-FileInfo-name {
+    font-weight: 600;
```

Comment:
we're almost evenly split on using `bold` (equivalent to `700`), and `600` directly. we can probably consolidate these but will need to check the BDL and possibly use a variable to describe it

Reply:
Okay. Is there something I need to change?

Reply:
From looking at src/elements/common/fonts.scss, looks like we only set the face and weight for `normal` and `bold`. My guess is we're meant to use those two, and not the number values